### ISSUE TITLE
Unable to move pawn forward two spaces on start

### ISSUE THREAD

**Issue by CalderMarshall-zz** [ryanheathers/react-chess#1]
Steps to recreate bug:

clone repo
npm install
npm start
go to  http://localhost:4000/
try to move a pawn two spaces

Game will not allow move

**Comment by ryanheathers** [comment]
Yep I don't think I built that in. Happy to take a PR though. I've had zero time to work on this since I did the proof-of-concept long ago, so there are definitely gaps in the gameplay.
